unable to initialize plugin.

```
13:44:24.823 [main] ERROR gitbucket.core.plugin.PluginRegistry - Error during plugin initialization: gitbucket-branch-protect-plugin_2.12-1.0.1.jar
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at gitbucket.core.plugin.PluginRegistry$.$anonfun$initialize$3(PluginRegistry.scala:291)
	at gitbucket.core.plugin.PluginRegistry$.$anonfun$initialize$3$adapted(PluginRegistry.scala:283)
	at scala.collection.immutable.List.foreach(List.scala:305)
	at gitbucket.core.plugin.PluginRegistry$.initialize(PluginRegistry.scala:283)
	at gitbucket.core.servlet.InitializeListener.$anonfun$contextInitialized$1(InitializeListener.scala:95)
	at gitbucket.core.servlet.InitializeListener.$anonfun$contextInitialized$1$adapted(InitializeListener.scala:63)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.$anonfun$withTransaction$2(BlockingProfile.scala:207)
	at slick.JdbcProfileBlockingSession$BlockingSession.withTransaction(TransactionalJdbcBackend.scala:26)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.$anonfun$withTransaction$1(BlockingProfile.scala:207)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.withSession(BlockingProfile.scala:200)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.withTransaction(BlockingProfile.scala:207)
	at gitbucket.core.servlet.InitializeListener.contextInitialized(InitializeListener.scala:63)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:930)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:553)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:889)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:356)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1445)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1409)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:822)
	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:275)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:100)
	at org.eclipse.jetty.server.handler.StatisticsHandler.doStart(StatisticsHandler.java:255)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.server.Server.start(Server.java:407)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:100)
	at org.eclipse.jetty.server.Server.doStart(Server.java:371)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at JettyLauncher.main(JettyLauncher.java:136)
Caused by: java.lang.NoSuchMethodError: scala.Predef$.wrapRefArray([Ljava/lang/Object;)Lscala/collection/mutable/WrappedArray;
	at Plugin.<init>(Plugin.scala:8)
	... 38 common frames omitted
13:44:24.829 [Thread-11] INFO  g.core.plugin.PluginWatchThread - Start PluginWatchThread: /home/aravind/.gitbucket/plugins
13
```

--- a/build.sbt
+++ b/build.sbt
@@ -2,12 +2,12 @@ name := "gitbucket-branch-protect-plugin"
 
 organization := "com.github.tohosokawa"
 
-version := "1.0.1"
+version := "1.1.0"
 
-scalaVersion := "2.12.2"
+scalaVersion := "2.13.3"
 
 libraryDependencies ++= Seq(
-  "io.github.gitbucket" %% "gitbucket"         % "4.16.0" % "provided",
+  "io.github.gitbucket" %% "gitbucket"         % "4.32.0" % "provided",
   "javax.servlet"        % "javax.servlet-api" % "3.1.0"  % "provided"
 )
 

--- a/src/main/scala/Plugin.scala
+++ b/src/main/scala/Plugin.scala
@@ -5,7 +5,12 @@ class Plugin extends gitbucket.core.plugin.Plugin {
   override val pluginId: String = "branch-protect"
   override val pluginName: String = "Branch Protect Plugin"
   override val description: String = "Disables force and non force-pushes to this branch and prevents it from being deleted."
-  override val versions: List[Version] = List(new Version("1.0.0"),new Version("1.0.1"))
+  override val versions: List[Version] = List(
+    // 4.15.0 - 4.31.2
+    new Version("1.0.0"),new Version("1.0.1"),
+    // 4.32.0 -
+    new Version("1.1.0")
+  )
 
   override val receiveHooks = Seq(
     new ProtectedBranchReceiveHook()

